2.process-data/2.0.process-eigenvalues.py

```diff
@@ -0,0 +1,137 @@
+# %% Initialization
+import sys
+from pathlib import Path
+
+import os
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from sklearn.decomposition import PCA
+from joblib import dump, load
+
+basefolder_loc = Path(__file__).parents[1]
+sys.path.append(str(basefolder_loc))
+from utils.load_image import get_all_images
+
+
+# %% load data
+def load_data(dataframe):
+    image_generator = get_all_images(dataframe)
+
+    all_images = []
+    all_targets = []
+    all_cell_codes = []
+    for data in image_generator:
+        # Crop image to center reduce the image to 25%
```

Comment:
Good point, I would like to make separate issue out of it.

In my view: big cells (bigger as 101x101) will loss information. But I don't think it will matter that much, but many did it because of memory usages